packages/README.md

```diff
@@ -0,0 +1,10 @@
+# Coding agreements in DevTools
+
+## ValueListeneble and ValueNotifier
+
+When an object owns and exposes a listenable value,
+we declare the related fields always in the same order:
+
+1. Public getter for ValueListeneble
+2. Private ValueNotifier
+3. Public setter
```

Comment:
Do we really surface a public setter? I thought instances of this patter were a public getter with no setter.

Reply:
hm. I cannot find example in the code, but it is what I remember Kenzie said to me some time ago.
@kenzieschmoll , did I recollect it right?